src/components/input/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {TextInput} from 'react-native';
+
+import {inputStyles} from './inputStyles';
+
+interface InputProps {
+  input?: string;
+  secureTextEntry?: boolean;
+  placeholder: string;
+  onChangeText: (input: string) => void;
+  value: string;
+  keyboardType?: string;
+}
+
+export const InputText: React.FC<InputProps> = props => (
```

Comment:
InputText - name is looks weird, also file name is not valid with component name ...
```
({style, ...props}) => (
<TextInput
  {...props}
  style={[inputStyles.inputContainer, style]}
/>
```